ci/docker/golang/Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM debian:buster-slim
+# TODO: Change base image to ubuntu:20.04 on 2024-06-30
```

Comment:
Specifying that this is the current `oldstable` should be enough

Reply:
It won't. The current `oldstable` is `bullseye`, but we need also to support `buster` which is now `oldoldstable`. Also, for example after 2026-06-30 the oldest maintained Debian will be `bookworm`, which contains libc 2.36, but we will still need to support Ubuntu 22.04 until 2027-06-01, which contains libc 2.35. So switching every year between the oldest supported Debian and Ubuntu versions is the only solution.